File: 88-merge-sorted-array/88-merge-sorted-array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        # make the arrays of the length provided
        # merge both arrays and sort them
        while len(nums1) != m:
            nums1.pop()
        nums1 += nums2
        nums1.sort()
        

File: 88-merge-sorted-array/88-merge-sorted-array.js
/**
 * @param {number[]} nums1
 * @param {number} m
 * @param {number[]} nums2
 * @param {number} n
 * @return {void} Do not return anything, modify nums1 in-place instead.
 */
var merge = function(nums1, m, nums2, n) {
    for (let i = 0; i<nums2.length; i++){
        nums1.push(nums2[i]);
    }
    let i = 0;
    while (i < (m+n)){
        if ((nums1[i] === 0) && (nums1.length !== m+n)){
            nums1.splice(i, 1);
        } else {
            i += 1;
        }
    }
    nums1.sort((a, b) => a - b);
}